xml/System.Buffers/SequenceReaderExtensions.xml

```diff
@@ -14,7 +14,7 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Provides extended functionality for the <see cref="T:System.Buffers.SequenceReader`1" /> class that allows parsing and reading of UTF-8, UTF-16, and binary data.</summary>
```

Comment:
The type definition is incorrect since it says `T : struct`. It should be `unmanaged`
![image](https://user-images.githubusercontent.com/6527137/62092630-b796e180-b22a-11e9-9376-93eab111d639.png)

Is this something that can be fixed?
https://github.com/dotnet/corefx/blob/fec76b0fc06b4fce9e4ac28b9a2bf1101209d984/src/System.Memory/src/System/Buffers/SequenceReader.cs#L12

Also, what about the other generic constraint that `T  implements IEquatable<T>`